### ISSUE TITLE
installation

### ISSUE THREAD

**Issue by Samanasuki** [MelvinGr/ProjectFaolan#2]
could you describe a brief instruction on how to start this server ?

**Comment by xazes** [comment]
whats the problem? you build it and start the server. for me it works. the only thing is you can only run in the world.

**Comment by Samanasuki** [comment]
the problem is that I downloaded the source code, I want to compile it,
but an error occurs, is there any compilation instructions ?

пн, 31 мая 2021 г. в 17:01, xazes ***@***.***>:

**Comment by xazes** [comment]
what errors come? i can try make tomorrow a compile instruction.

**Comment by Samanasuki** [comment]
[image: изображение.png]

Severity Code Description Project File String Suppression Status Error
MSB3073 exiting the ""command
C:\Users\arzamasovvo\.nuget\packages\gitversion.msbuild\5.6.6\tools\net48/gitversion.exe"
"C:\Users\arzamasovvo\Desktop\ProjectFaolan-master\Faolan.Core" -output
file -outputfile obj\gitversion.json" with the code 1. Faolan.Core
C:\Users\arzamasovvo\.nuget\packages\gitversion.msbuild\5.6.6\tools\GitVersion.MsBuild.targets
9


вт, 1 июн. 2021 г. в 02:10, xazes ***@***.***>:

**Comment by xazes** [comment]
yes you dont have import it via git in visual studio. make a new project with directly git import. or commant out the git commands in code.

**Comment by Samanasuki** [comment]
Thank you, it's loaded, can you tell me how [image: изображение.png]
to create an account ? and how to get into the database ?

вт, 1 июн. 2021 г. в 23:44, xazes ***@***.***>:

**Comment by Samanasuki** [comment]
and where to get a client ?

ср, 2 июн. 2021 г. в 11:28, Василий Арзамасов ***@***.***>:

**Comment by xazes** [comment]
you cant create account or anything. its only walk in world server from @MelvinGr . for db you can use any sqlite browser.

**Comment by Hacknir** [comment]
I dont know how to PM directly to MelvinGr but i must say that there are many people who interested in this server, expecially if he create server equal to 1.4-1.5 patch of Age of Conan, pre-Khitai expansion. 100+ and maybe more waiting for that one

**Comment by xazes** [comment]
yes in the past i has try to contact melvin too. we can only wait that he comes back.

**Comment by Hacknir** [comment]
> yes in the past i has try to contact melvin too. we can only wait that he comes back.

perhaps we will never see him again...10+ month have passed

**Comment by xazes** [comment]
Melvin is active 🤗👏🎉🎉🎉🎉

**Comment by ImmortalusDev** [comment]
If emulator will be converted to C++, i'm pretty sure there will be more people for development.
